.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -31,9 +31,9 @@ body:
       label: What version of Home Assistant Core has the issue?
       placeholder: core-
       description: >
-        Can be found in: [Settings -> About](https://my.home-assistant.io/redirect/info/).
+        Can be found in: [Settings ⇒ System ⇒ Repairs ⇒ Three Dots in Upper Right ⇒ System information](https://my.home-assistant.io/redirect/system_health/).
```

Comment:
I think we should point to the about page and not send people in this deep.

Reply:
@frenck I think the proposal is for the user to be able to answer both the 1st and 2nd version questions using the same link.

I.e. they visit one page not two.

Reply:
Exactly. Besides that, the about page doesn't show the version with the`core-` prefix, so it's a tad confusing as well.

Reply:
Yeah that is true 👍